<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistMainWindow</class>
 <widget class="QMainWindow" name="RegistMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="RegistAgainPasswordlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string>Password Again:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RegistRegistpushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RegistIDlabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string>Enter ID：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RegistAgainPasswordlineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="RegistPasswordlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string>Enter Password：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RegistPasswordlineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="RegistIDlineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="RegistWavelabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>338</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RegistWarnlabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Too Short！At Least 6！</string>
    </property>
   </widget>
   <zorder>RegistWavelabel</zorder>
   <zorder>RegistAgainPasswordlabel</zorder>
   <zorder>RegistRegistpushButton</zorder>
   <zorder>RegistIDlabel</zorder>
   <zorder>RegistAgainPasswordlineEdit</zorder>
   <zorder>RegistPasswordlabel</zorder>
   <zorder>RegistPasswordlineEdit</zorder>
   <zorder>RegistIDlineEdit</zorder>
   <zorder>RegistWarnlabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRegist">
    <property name="title">
     <string>Regist</string>
    </property>
   </widget>
   <addaction name="menuRegist"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
